props/saber_fett263_buttons.h

```diff
@@ -257,7 +325,100 @@ SaberFett263Buttons() : PropBase() {}
       case EVENTID(BUTTON_POWER, EVENT_CLICK_SHORT, MODE_OFF | BUTTON_AUX):
         previous_preset();
         return true;
+
+       // Gesture Controls
+		
+       // Battle Mode
+		    
+       #ifdef FETT263_BATTLE_MODE
+       case EVENTID(BUTTON_NONE, EVENT_SWING, MODE_ON | BUTTON_AUX):
+           if (!battle_mode_) {
+              battle_mode_ = true;
+              hybrid_font.SB_Force();
+           } else {
+              battle_mode_ = false;
+              hybrid_font.SB_Force();
+           }
+           return true;
+
+       // Auto Lockup Mode
+       case EVENTID(BUTTON_NONE, EVENT_CLASH, MODE_ON):
+           if (battle_mode_) {
+              clash_impact_millis_ = millis();
+              swing_blast_ = false;
+              if (clash_impact_millis_ > last_swing_) {
+                 SaberBase::SetLockup(SaberBase::LOCKUP_NORMAL);
+                 swing_blast_ = false;
+                 auto_lockup_on_ = true;
+                 SaberBase::DoBeginLockup();
+                 return true;
+               }
+             } else {
+               return false;
+           }
+
+       case EVENTID(BUTTON_NONE, EVENT_STAB, MODE_ON):
+           if (battle_mode_) {
+              clash_impact_millis_ = millis();
+              swing_blast_ = false;
+              if (clash_impact_millis_ > last_swing_) {
+                 if ((fusor.angle1() + M_PI / 2) * (32768 / M_PI) < 8000) {
```

Comment:
I think this should be:

  if (fuser.angle1() < - M_PI / 2) {

Reply:
Yeah I could use your help on the math here, it's probably "ham-fisted" but it worked.  Essentially I only want Drag if the blade is 0 to 45 degrees (approximately), to allow for slightly downward Melt actions to be used.  This equation produces the control I want but if there's a more efficient way, I believe M_PI / 2 would be 90 degrees so would M_PI/4 be 45 degrees? The angle stuff has been too long since I used in high school so it still trips me up.

Reply:
M_PI /2 is 45 degrees

Reply:
perfect then, thank you

Reply:
Not sure why but this does not work as expected, the previous equation worked perfectly with fusor.angle1() < -M_PI / 2 I cannot trigger Drag at all. reverted to original and it runs as expected with Drag only triggering if pointing straight down or up 8:00, I've changed everything else and it's all running, this change did not work so I left out.

Reply:
My fault, you were right, it should be M_PI / 4

Reply:
forgot to update, I changed to 

fusor.angle1() < - M_PI / 4 

and it's working, updated the file